src/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
```

Comment:
This looks good! Great to see you added the `key` to each element as you iterated through the array.

A very small suggestion. Since this is a footer component on the page, it would make semantic sense for you to wrap the `<ul>` element inside a `<footer>` element and move the class onto this new `<footer>` element. It won't change any visually on the screen, but improve the accessibility of your page (screen readers for people who are blind rely on semantics in pages to describe the page better for them).